datashader/glyphs/line.py

```diff
@@ -1107,18 +1152,13 @@ def cpu_antialias_2agg_impl(sx, tx, sy, ty, xmin, xmax, ymin, ymax, xs, ys,
             if ncols == 1:
                 return
 
-            if j == 0:
-                accum_aggs = [aggs[k].copy() for k in range(n_aggs)]
-            else:
-                for k in range(n_aggs):
-                    _combine_in_place(accum_aggs[k], aggs[k], antialias_combinations[k])
+            _aa_stage_2_accumulate(aggs_and_accums, antialias_combinations)
```

Comment:
Note the increase in readability of this function and others like it by using separate jit-able functions like `_aa_stage_2_accumulate`. This also allows functions like this to have their `@expand_aggs_and_cols` decorators reinstated, in line with the other various low-level line rendering functions.